### DISCUSSION TITLE
[libz-sys] Can the upload|download-artifact version 4.0 actions break linux cross installations on CI?

### DISCUSSION THREAD

**Discussion by Byron** [cross-rs/cross#1387] [Q&A] [answered]
I am having a curious issue where dependabot updates of both [upload-artifact](https://github.com/rust-lang/libz-sys/pull/163) and [download-artifact](https://github.com/rust-lang/libz-sys/pull/164) break [the installation of cross](https://github.com/rust-lang/libz-sys/actions/runs/7245060469/job/19734395848?pr=163) - but only on Linux?

What's particularly strange is that *the download-artifact step fails even in the PR updating the upload-artifact action*.

Would you have any advise in the matter?

Thanks for your help.

**Comment by Emilgardis** [comment, marked as answer]
Not a cross issue! See https://github.com/actions/download-artifact/issues/251
